lib/internal/test_runner/coverage.js

```diff
@@ -177,6 +187,13 @@ class TestCoverage {
         if (j > 0 && ranges.length > 0) {
           const range = ranges[0];
 
+          ArrayPrototypePush(functionReports, {
+            __proto__: null,
+            name: functions[j].functionName,
+            count: MathMax(...ArrayPrototypeMap(ranges, (r) => r.count)),
```

Comment:
Nit: you already go over the range items in line 166, instead of recalculating the max here, you can just recalculate it there